README.md

```diff
@@ -290,6 +294,14 @@ global:
       some_int: 1234
       some_float: 3.14159
       some_bool: false
+    key-mapping:
+      # key-mapping is used to replace the attribute key with different keys
+      - key: servertracer.http.responsecode
+        replacement: http.status_code
+      - key:  servertracer.http.responsephrase
+        replacement: http.message
+        overwrite: true # replace attribute key even if the replacement string is already a key on the span attributes
+        keep: true # keep the attribute with the original key
```

Comment:
Nit: `keep_original` may be clearer.

Reply:
I'm going to keep the shorter variations, I think people will need to check the readme anyway for this.